.buildbot.sh

```diff
@@ -12,6 +12,6 @@ export PATH=`pwd`/.cargo/bin/:$PATH
 
 rustup component add --toolchain nightly rustfmt-preview || cargo +nightly install --force rustfmt-nightly
 
-cargo +nightly fmt --all -- --check
+cargo +nightly fmt --all -- --check gc_tests/tests/*
```

Comment:
Does this mean we *only* check `gc_tests/tests/*`?

Reply:
No I've tested it, it's in addition to `-all`.

Reply:
OK.